Src/Sarif.PatternMatcher/JsonLogicalLocationProcessor.cs

```diff
@@ -27,6 +67,9 @@ public class JsonLogicalLocationProcessor
         ///     - For arrays, one might report the whole array, but this gets "closer" to the match.
         ///     - For objects, spanning properties will get "closer" to the match.
         /// </remarks>
+        /// <param name="results">collection of JSON results to process.</param>
```

Comment:
>collection [](http://example.com/codeflow?start=34&length=10)

I don't think that most doc comments of this kind are sentence fragments. Please review actual .NET docs comments to get a sense of what the standard is.

Reply:
Improved the XML comments.
Aware that typical comments are more complete.
Had to stop to study code to get a sense of something more appropriate to write.
Trying to balance where to spend time.